### DISCUSSION TITLE
Problem after setup unable to load but able to load --- layout: home # Index page ---

### DISCUSSION THREAD

**Discussion by muizyasin** [cotes2020/jekyll-theme-chirpy#1992] [Q&A]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

I try many time, plenty step, tutorial online and tips and trick still unable to solve this issues. but if i install starter pack i able to run just its content empty.. 
![image](https://github.com/user-attachments/assets/a19c3c38-26da-49ab-a6d7-a81f356e111e)

i already try edit .yml file still same result. i even want try to re run for deploy but seems stuck at queue
![image](https://github.com/user-attachments/assets/6820f336-2c83-4962-bd3e-ce476c3c3e8f)

### Operations you have already tried

yup true already plenty raise this problem and i also already tried all solution but still no good outcomes.

**Comment by kungfux** [comment]
It should take you a few clicks as described in the Getting Started guide under [Option 1. Using the Starter (Recommended)](https://chirpy.cotes.page/posts/getting-started/#option-1-using-the-starter-recommended).

**Reply by muizyasin** [reply]
for the starter i can apply fine but for customization i unable to make it run in host

**Reply by kungfux** [reply]
Double check your build action. You can refer to [this](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/.github/workflows/ci.yml) and [this](https://github.com/cotes2020/chirpy-starter/blob/main/.github/workflows/pages-deploy.yml) one as a reference.
